src/android/HockeyApp.java

```diff
@@ -115,6 +180,55 @@ public void run() {
             return true;
         }
         
+        if (action.equals("feedbackModal")) {
+            final ArrayList<Uri> attachments = new ArrayList<Uri>();
+            final Activity context = cordova.getActivity();
+            boolean takeScreenshot = args.optBoolean(0);
+            if (takeScreenshot) {
+                Bitmap screenshot = getBitmap();
```

Comment:
Good point. I looked into that option when I first wrote this function and decided not to use it mainly for 2 reasons:

1. It would not work with Crosswalk web views because they have their own built-in method for capturing a screenshot, afaik `View.getDrawingCache()` would not work properly.
2. It leaves traces of the screen shot in the external storage of the device and the picture gallery. The name of the file in the device is not easy to determine either.